examples/ring_app/src/examples/ring_app/repl.clj

```diff
@@ -0,0 +1,86 @@
+(ns examples.ring-app.repl
```

Comment:
I added this line to the project.clj in df5c156:

```clj
:repl-options {:init-ns examples.ring-app.repl}
```

This allows for running `lein repl` to load up the ring-app repl namespace automatically.  That way, you'd only need to run `(go)` after the first prompt comes up in order to startup the app.  I also added a brief blurb about this to the README doc.